SPECS/curl.spec

```diff
@@ -202,7 +271,7 @@ sed -e 's|^35$|35,52|' -i tests/data/test323
     eval "$cmd"
 )
 
-%if 0%{?xenserver} > 8
+%if %{with xcpng}
```

Comment:
Should be "without", at least in this first commit? The previous condition evaluated to false in the XS version of the spec file.

Reply:
Right, this seems to have to do with the nss configuration flag